instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/propagators/PropagationType.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.propagators;
+
+/** Types of supported propagators. */
+public enum PropagationType {
+
+  /** B3Propagator */
+  B3,
+
+  /** W3CTraceContextPropagator */
+  W3C,
+
+  /** W3CBaggagePropagator */
+  BAGGAGE,
+
+  /** JaegerPropagator */
+  JAEGER,
+
+  /** OtTracePropagator */
+  OT_TRACER,
+
+  /** NoOp propagator */
+  NOOP
```

Comment:
I don't think you need that option, if you want no-op you can just set `enabled = false`.